allauth/account/forms.py

```diff
@@ -462,9 +462,13 @@ def __init__(self, *args, **kwargs):
         self.fields['password1'].user = self.user
 
     def clean_oldpassword(self):
-        if not self.user.check_password(self.cleaned_data.get("oldpassword")):
+        old_password = self.cleaned_data.get("oldpassword")
+        if not old_password:
             raise forms.ValidationError(_("Please type your current"
                                           " password."))
+        if not self.user.check_password(old_password):
+            raise forms.ValidationError(_("The provided password is "
```

Comment:
Sorry for nitpicking on details. Elsewhere, this style is used:

"The username and/or password you specified are not correct."

Can you change this error message to:

"The password you specified is not correct."

Reply:
Sorry for the delay. I am quite new to this stuff - is there a way to make the suggested changes in the merge request directly or is it better to create a new merge request from a different branch altogether?